### DISCUSSION TITLE
New project and questions. Supply Chain and Rain Concern.

### DISCUSSION THREAD

**Discussion by rcowan75** [nebhead/PiFire#89] [Q&A]
I just picked up a broken pellet smoker tonight with the plans of fixing it with a PiFire! However, I went to order and Zero w 2 and was surprised to find everyone is out of them! Bummer. I should have checked that first. Has this been a problem for a while? Any suggestions on where to search?
Second, I'm in Oregon and it rains.... often. Has anyone done anything about weather proofing a PiFire? It can be covered when not in use, but it's hard to assume no rain for a lot of time here. I'm thinking of maybe designing a rain hood for the already designed control panel housing but open to ideas. Thanks!

**Comment by nebhead** [comment]
Oh man, supply chain issues are a bummer.  I'm actually still running my Pi
Zero W 1 on my grill. So if you can get your hands on one of those you can
use that until you can find a 2.  Or just keep running the 1.

I grew up in Oregon so I know what you mean!  For rain proofing, I think
you'll probably just need to design a cover for the top of the screen
area.  The rest is pretty well protected.  Obviously it's best to have a
full cover for your grill, but in case you forget to put that back on, it
would hopefully be good enough.

**Comment by rcowan75** [comment]
I designed a rain cover for the Display Housing. It sits over the screen. I can't read the screen while it's on but it lifts off easily to read. It's basically untested but I wanted to upload before I forget to incase it's helpful to someone else. :-)

https://www.thingiverse.com/thing:5396648
